php-nginx/php.ini

```diff
@@ -478,7 +478,7 @@ error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT
 ; Development Value: On
 ; Production Value: Off
 ; http://php.net/display-errors
-display_errors = Off
+display_errors = stderr
```

Comment:
According to http://php.net/manual/en/errorfunc.configuration.php#ini.display-errors, `display_errors` should not be used on production systems.

Is it really OK to turn it on?